src/joint_types.jl

```diff
@@ -14,7 +14,10 @@ function _local_coordinates!(jt::JointType,
 end
 
 function _global_coordinates!(jt::JointType, q::AbstractVector, q0::AbstractVector, ϕ::AbstractVector)
-    q .= q0 .+ ϕ # TODO: allocates on 0.5
```

Comment:
```julia
@benchmark x .= (+).(y, z) setup = (x = rand(10); y = rand(10); z = rand(10))
```

still shows 32 bytes of allocation (independent of size of vectors). It's better than `x .= y .+ z` though.